src/contacts.cs

```diff
@@ -463,16 +1458,30 @@ interface CNContactStore {
 
 		[Export ("unifiedContactsMatchingPredicate:keysToFetch:error:")]
 		[Protected] // we cannot use ICNKeyDescriptor as Apple (and others) can adopt it from categories
-		CNContact[] GetUnifiedContacts (NSPredicate predicate, NSArray keys, out NSError error);
+		[return: NullAllowed]
+		CNContact[] GetUnifiedContacts (NSPredicate predicate, NSArray keys, [NullAllowed] out NSError error);
```

Comment:
What about `NSObject [] keys`? Nicer C# also for the ones below, @spouliot what do you think?

Reply:
Same: why not `ICNKeyDescriptor[]`?

Reply:
@dalexsoto and I looked at this one...it appears that a lot of the bindings were written with a custom wrapper `CNContact` for `ICNKeyDescriptor[]`. I am leaving the old bindings with the wrapper as updating them would be a breaking change, but for new bindings using `ICNKeyDescriptor[]`.